src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Guarantees: immutable
+ */
+public class Contact {
+
+    public String value;
+    boolean isPrivate;
```

Comment:
It's a good practice to limit all class members permission whenever possible.
Consider setting permission to private